.eslintignore

```diff
@@ -0,0 +1,8 @@
+*.js
+*.js.map
+*.d.ts
+
+node_modules
+lib
+samples
```

Comment:
maybe we should also lint our samples? So we present "pretty" code.

Reply:
Done